src/Sentry/Internal/Hub.cs

```diff
@@ -196,6 +196,9 @@ internal ITransactionTracer StartTransaction(
         sampleRate ??= _options.TracesSampleRate ?? 0.0;
         isSampled ??= context.IsSampled ?? SampleRandHelper.IsSampled(sampleRand, sampleRate.Value);
 
+        // Ensure the actual sampleRate is set on the provided DSC (https://github.com/getsentry/team-sdks/issues/117)
```

Comment:
todo: when an explicit sampling decision is forced
- when `true`, set DSC `sample_rate` to `1.0`
- when `false`, set DSC `sample_rate` to `0.0`

Reply:
When the `TracesSampler` is not applied,
and a sampling decision has been made upstream (distributed tracing) already,
then
- setting DSC `sample_rate` to `1.0` when upstream decision was `true`
- setting DSC `sample_rate` to `0.0` when upstream decision was `false`

Reply:
This logic has been removed:
The final behavior of this changeset is:

When the TracesSampler is not applied,
and a sampling decision has been made upstream (distributed tracing) already,
then the DSC `sample_rate` **is not** updated/overwritten.